fe/fe-core/src/main/java/com/starrocks/privilege/PrivilegeManager.java

```diff
@@ -42,6 +43,11 @@ public class PrivilegeManager {
     private static final String ALL_ACTIONS = "ALL";
     // builtin roles and users
     private static final String ROOT_ROLE_NAME = "root";
+    public static final long ROOT_ROLE_ID = -1;
```

Comment:
ignore, can change it next pr

Reply:
this field must be public because AuthUpgrader will use it.

Reply:
I didn't found it in AuthUpgrader, currently the built-in role ids are only used in PrivilegeManager

Reply:
It will be used. A user with admin_priv/grant_priv/node_priv in older version will be granted to built-in roles in newer version. This PR only support upgrading select & impersonate.

Reply:
got it